README.md

```diff
@@ -20,7 +20,20 @@
 <p>Lux is a command-line interface for controlling and monitoring Govee lighting strips built in Go. Lux provides it's users with the ability to manage their lighting strips from their desktop computer from any network with an internet connection.</p>
 
 <h2 id="installation">Installation</h2>
-<p>Currently, to install Lux, you must download the source and compile it on your own. After releases are made, we will provide & host various executables.</p>
+<p>We provide a simple installer for each release for Windows but if that's not suitable for you, you can easily compile Lux directly from source. </p>
+<p>You can attain the latest copy of the Lux installer from our <a href="https://github.com/BanDev/Lux/releases">releases</a> page on GitHub, or through <a href="https://docs.microsoft.com/en-us/windows/package-manager/winget/">Winget</a> using the command below:</p>
+
+<pre>
+winget install BanDev.Lux
+</pre>
+
+<blockquote>If you choose to self compile, you will need to add the path to `lux.exe` to your `Path` environment variables in order for Lux to be detected as a command in PowerShell & CMD.</blockquote>
+
+<p>If you're already using a terminal, restart it to update environment variables and then run the following to begin setting up your instance of Lux.</p>
```

Comment:
```suggestion
<p>If you're already using a terminal, restart it so that it can register the changes to the environment variables. Then, run the following command to begin setting up your instance of Lux.</p>
```